.github/workflows/stable.yml

```diff
@@ -149,31 +149,34 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ env.TOOLCHAIN }}
+          toolchain: ${{ env.TOOLCHAIN }}-${{ matrix.target }}
```

Comment:
Do we need this? There is `target:` in the next line.

Reply:
The CI logs from this PR and other commit suggest divergences:
```
C:\Users\runneradmin\.cargo\bin\rustup.exe toolchain install stable-x86_64-pc-windows-gnu
info: syncing channel updates for 'stable-x86_64-pc-windows-gnu'
info: latest update on 2022-06-30, rust version 1.62.0 (a8314ef7d 2022-06-27)
info: downloading component 'cargo'
info: downloading component 'rust-mingw'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: installing component 'cargo'
info: installing component 'rust-mingw'
info: installing component 'rust-std'
info: installing component 'rustc'

  stable-x86_64-pc-windows-gnu installed - rustc 1.62.0 (a8314ef7d 2022-06-27)
```
```
C:\Users\runneradmin\.cargo\bin\rustup.exe toolchain install stable
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'

  stable-x86_64-pc-windows-msvc unchanged - rustc 1.62.0 (a8314ef7d 2022-06-27)
```